### PR TITLE
resource_control: add more info about sql ru consumption

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/16007
- Other reference link(s):https://github.com/pingcap/docs/pull/15934

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by glorv — COMMENTED** [review]
LGTM

File: tidb-resource-control.md [hunk 1]
```diff
@@ -522,6 +518,85 @@ By default, the task types that are marked as background tasks are `""`, and the
...
+
+    `ru_consumption` in the result is the RU consumed by the execution of this SQL statement.
+
+#### `EXPLAIN ANALYZE`
+
+You can use the [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md#ru-request-unit-consumption) statement to get the amount of RUs consumed during SQL execution. Note that the amount of RUs is affected by the cache (for example, [coprocessor cache](/coprocessor-cache.md)). When the same SQL is executed multiple times, the amount of RUs consumed by each execution might be different. The RU value does not represent the exact value for each execution, but can be used as a reference for estimation.
+
+#### Slow queries and corresponding system table
+
+When you enable resource control, the [slow query log](/identify-slow-queries.md) of TiDB and the corresponding system table [`INFORMATION_SCHEMA.SLOW_QUERY`](/information-schema/information-schema-slow-query.md) contain the resource group and RU consumption of the corresponding SQL.
```
Comment (songrijie) [inline tidb-resource-control.md:567]:
```suggestion
When you enable resource control, the [slow query log](/identify-slow-queries.md) of TiDB and the corresponding system table [`INFORMATION_SCHEMA.SLOW_QUERY`](/information-schema/information-schema-slow-query.md) contain the resource group, RU consumption of the corresponding SQL and the time spent waiting for available RUs.
```

**Review by songrijie — APPROVED** [review]
rest LGTM

File: tidb-resource-control.md [hunk 2]
```diff
@@ -522,6 +518,85 @@ By default, the task types that are marked as background tasks are `""`, and the
...
+    ```
+    +------------------------------------------------------------------------------------------------------------------------+
+    | @@tidb_last_query_info                                                                                                 |
+    +------------------------------------------------------------------------------------------------------------------------+
+    | {"txn_scope":"global","start_ts":446809472210829315,"for_update_ts":446809472210829315,"ru_consumption":4.34885578125} |
+    +------------------------------------------------------------------------------------------------------------------------+
+    1 row in set (0.01 sec)
+    ```
+
+    `ru_consumption` in the result is the RU consumed by the execution of this SQL statement.
```
Comment (hfxsd) [inline tidb-resource-control.md:559]:
```suggestion
    In the result, `ru_consumption` is the RU consumed by the execution of this SQL statement.
```

File: tidb-resource-control.md [hunk 3]
```diff
@@ -522,6 +518,85 @@ By default, the task types that are marked as background tasks are `""`, and the
...
+
+#### `EXPLAIN ANALYZE`
+
+You can use the [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md#ru-request-unit-consumption) statement to get the amount of RUs consumed during SQL execution. Note that the amount of RUs is affected by the cache (for example, [coprocessor cache](/coprocessor-cache.md)). When the same SQL is executed multiple times, the amount of RUs consumed by each execution might be different. The RU value does not represent the exact value for each execution, but can be used as a reference for estimation.
+
+#### Slow queries and corresponding system table
+
+When you enable resource control, the [slow query log](/identify-slow-queries.md) of TiDB and the corresponding system table [`INFORMATION_SCHEMA.SLOW_QUERY`](/information-schema/information-schema-slow-query.md) contain the resource group, RU consumption of the corresponding SQL and the time spent waiting for available RUs.
```
Comment (hfxsd) [inline tidb-resource-control.md:567]:
```suggestion
When you enable resource control, the [slow query log](/identify-slow-queries.md) of TiDB and the corresponding system table [`INFORMATION_SCHEMA.SLOW_QUERY`](/information-schema/information-schema-slow-query.md) contain the resource group, RU consumption of the corresponding SQL, and the time spent waiting for available RUs.
```

File: tidb-resource-control.md [hunk 4]
```diff
@@ -522,6 +518,92 @@ By default, the task types that are marked as background tasks are `""`, and the
 
 </CustomContent>
 
+## Observability of RU consumption
```
Comment (ran-huang) [inline tidb-resource-control.md:521]:
```suggestion
## View RU consumption
```

File: tidb-resource-control.md [hunk 5]
```diff
@@ -522,6 +518,92 @@ By default, the task types that are marked as background tasks are `""`, and the
...
+    1 row in set (0.01 sec)
+    ```
+
+    In the result, `ru_consumption` is the RUs consumed by the execution of this SQL statement.
+
+#### View RUs consumed during SQL execution by `EXPLAIN ANALYZE`
+
+You can use the [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md#ru-request-unit-consumption) statement to get the amount of RUs consumed during SQL execution. Note that the amount of RUs is affected by the cache (for example, [coprocessor cache](/coprocessor-cache.md)). When the same SQL is executed multiple times, the amount of RUs consumed by each execution might be different. The RU value does not represent the exact value for each execution, but can be used as a reference for estimation.
+
+#### Slow queries and corresponding system table
```
Comment (ran-huang) [inline tidb-resource-control.md:572]:
```suggestion
#### Slow queries and the corresponding system table
```

**Review by ran-huang — APPROVED** [review]

**Review by glorv — APPROVED** [review]

File: tidb-resource-control.md [hunk 6]
```diff
@@ -522,6 +518,102 @@ By default, the task types that are marked as background tasks are `""`, and the
...
+
+<CustomContent platform="tidb">
+
+When you enable resource control, the [slow query log](/identify-slow-queries.md) of TiDB and the corresponding system table [`INFORMATION_SCHEMA.SLOW_QUERY`](/information-schema/information-schema-slow-query.md) contain the resource group, RU consumption of the corresponding SQL, and the time spent waiting for available RUs.
+
+</CustomContent>
+
+<CustomContent platform="tidb-cloud">
+
+When you enable resource control, the system table [`INFORMATION_SCHEMA.SLOW_QUERY`](/information-schema/information-schema-slow-query.md) contain the resource group, RU consumption of the corresponding SQL, and the time spent waiting for available RUs.
```
Comment (qiancai) [inline tidb-resource-control.md:582]:
```suggestion
When you enable resource control, the system table [`INFORMATION_SCHEMA.SLOW_QUERY`](/information-schema/information-schema-slow-query.md) contains the resource group, RU consumption of the corresponding SQL, and the time spent waiting for available RUs.
```

**Pull request merged**